### PR TITLE
fix(sdk): hatchery step log delay numeric

### PR DESCRIPTION
**Pull request by richardlt**
Signed-off-by: richardlt <richard.le.terrier@gmail.com>

1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#18529.1 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔ 
* Package
  * Docker ✘

**Review by yesnault — APPROVED** [review]

File: sdk/hatchery/hatchery.go [hunk 1]
```diff
@@ -595,8 +595,8 @@ func SendSpawnInfo(ctx context.Context, h Interface, jobID int64, spawnMsg sdk.S
 
 func logStepInfo(ctx context.Context, step string, queued time.Time) {
 	if id := ctx.Value(LogFieldJobID); id != nil {
-		ctx = context.WithValue(ctx, LogFieldJobStep, step)
-		ctx = context.WithValue(ctx, LogFieldJobStepDelay, time.Since(queued).Milliseconds())
+		ctx = context.WithValue(ctx, LogFieldStep, step)
+		ctx = context.WithValue(ctx, LogFieldStepDelay, time.Since(queued).Nanoseconds())
```
Comment (fsamin) [inline sdk/hatchery/hatchery.go:599]: why do you want nano seconds, IMO it's not super easy to read as a humain

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#18529.2 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔ 
* Package
  * Docker ✘

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#18529.3 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔ 
* Package
  * Docker ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘

File: sdk/hatchery/hatchery.go (hunk 1 above)
Comment (richardlt) [reply sdk/hatchery/hatchery.go:599]: To be coherent with "latency" field.

**Pull request merged**